### PR TITLE
Feature/brp 89/application address

### PR DESCRIPTION
**Pull request by JoeChapman**
When a user has selected "Yes" I applied from within the UK on the first step of the /correct-mistakes journey
Then on step three they will be asked "Is your address the same as the address on the delivery letter?"
And if they select "No"
Then they will be asked to add the address from which they applied
And if the they select "Yes"
Then they will be redirected step four.

This PR makes the necessary changes to fulfil the above requirements.

The new application address, if entered, will be shown on the confirm step.

### CODE REVIEW

File: apps/correct-mistakes/index.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ router.use(mixins(fields, {
 }));
 
 router.use('/correct-mistakes/', wizard(require('./steps'), fields, {
-  controller: require('../../lib/base-controller'),
+  controller: hof.controllers.base,
```
Comment (easternbloc) [inline apps/correct-mistakes/index.js]: :ok_hand:

**Comment by easternbloc** [comment]
Are any changes required to the emails for same-address?

**Comment by JoeChapman** [comment]
@easternbloc to answer your question regarding the email, frankly, I don't know. One would assume that the additional info should be added to the email. You might have noticed that the fields for the address if it is different are not the contact address fields, therefore, the values are not automatically assigned to placeholders in the emails.

**Comment by Ben-Stevens** [comment]
Yes, the address should be played back to the applicant in the email.
Emails are now set to come to my inbox from dev (thanks Ben). So I can review them.

**Comment by JoeChapman** [comment]
@Ben-Stevens @easternbloc I have made the changes to the emails

**Pull request merged**